### PR TITLE
Remove Burn and Mint

### LINKED ISSUE #70: Remove Burn and Mint
PR https://github.com/many-protocol/specification/pull/20 removed Burn and Mint from the specification.

We should remove Burn and Mint from `many-framework`.

### PR DESCRIPTION
**Pull request by fmorency**
## Description

Remove Burn and Mint

Reference to `minters` were kept in LedgerStorage for future
reactivation

Depends on https://github.com/the-lifted-initiative/many-rs/pull/66

## Related Issue

Fixes #70

## Testing

- `cargo test`
- `run.sh`

## Breaking Change

- [x] Yes
- [ ] No

## Checklist:

- [x] I have read and followed the CONTRIBUTING guidelines for this project.
- [x] I have added or updated tests and they pass.
- [x] I have added or updated documentation and it is accurate.
- [x] I have noted any breaking changes in this module or downstream modules.

### CODE REVIEW

**Comment by fmorency** [comment]
CI will fail until https://github.com/the-lifted-initiative/many-rs/pull/66 is merged and this PR updated.

File: src/many-abci/src/abci_app.rs [hunk 1]
```diff
@@ -118,8 +118,7 @@ impl Application for AbciApp {
     fn begin_block(&self, request: RequestBeginBlock) -> ResponseBeginBlock {
         let time = request
             .header
-            .map(|x| x.time.map(|x| x.seconds as u64))
-            .flatten();
+            .and_then(|x| x.time.map(|x| x.seconds as u64));
```
Comment (hansl) [inline src/many-abci/src/abci_app.rs:121]: Ohhh that feels good on monday morning.

**Review by hansl — APPROVED** [review]
LGTM. Nice cleanup.

File: src/many-ledger/src/storage.rs [hunk 2]
```diff
@@ -264,88 +268,47 @@ impl LedgerStorage {
...
-            amount,
-        ));
-
-        if !self.blockchain {
-            self.persistent_store.commit(&[]).unwrap();
-        }
-
-        Ok(())
-    }
+    // TODO: Re-enable this when Minting is implemented
```
Comment (hansl) [inline src/many-ledger/src/storage.rs:271]: Actually you should remove the code entirely. Minting might never work this way.

**Review by hansl — APPROVED** [review]

**Pull request merged**